src/lib/actions/RulesAction.ts

```diff
@@ -1,12 +1,13 @@
-import {CodeAnalyzer, CodeAnalyzerConfig, Rule, RuleSelection} from '@salesforce/code-analyzer-core';
-import {CodeAnalyzerConfigFactory} from '../factories/CodeAnalyzerConfigFactory';
-import {EnginePluginsFactory} from '../factories/EnginePluginsFactory';
-import {createWorkspace} from '../utils/WorkspaceUtil';
-import {ProgressEventListener} from '../listeners/ProgressEventListener';
-import {LogFileWriter} from '../writers/LogWriter';
-import {LogEventListener, LogEventLogger} from '../listeners/LogEventListener';
-import {RuleViewer} from '../viewers/RuleViewer';
-import {RulesActionSummaryViewer} from '../viewers/ActionSummaryViewer';
+import { CodeAnalyzer, CodeAnalyzerConfig, Rule, RuleSelection } from '@salesforce/code-analyzer-core';
```

Comment:
In my heart of hearts, I don't actually care how we style our imports. But we should do our best to keep our IDEs' auto-formatting of imports consistent so they're not constantly re-styling each other's changes.

Reply:
Agreed, this is happening for me on default whenever I organize the imports in VS Code. Do you know if the IntelliJ default wants to keep the brackets cozy?